stdlib/public/Concurrency/AsyncStream.swift

```diff
@@ -141,22 +215,46 @@ public struct AsyncStream<Element> {
 
   let produce: () async -> Element?
 
-  /// Construct a AsyncStream buffering given an Element type.
-  ///
-  /// - Parameter elementType: The type the AsyncStream will produce.
-  /// - Parameter maxBufferedElements: The maximum number of elements to
-  ///   hold in the buffer past any checks for continuations being resumed.
-  /// - Parameter build: The work associated with yielding values to the 
-  ///   AsyncStream.
-  ///
-  /// The maximum number of pending elements limited by dropping the oldest
-  /// value when a new value comes in if the buffer would excede the limit
-  /// placed upon it. By default this limit is unlimited.
-  ///
-  /// The build closure passes in a Continuation which can be used in
-  /// concurrent contexts. It is thread safe to send and finish; all calls are
-  /// to the continuation are serialized, however calling this from multiple
-  /// concurrent contexts could result in out of order delivery.
+  /// Constructs an asynchronous stream for an element type, using the
+  /// specified buffering policy and element-producing closure.
+  ///
+  /// - Parameter elementType: The type of element the `AsyncStream`
+  ///   produces
```

Comment:
```suggestion
  ///   produces.
```

Trailing period to match the other parameters.